app/assets/javascripts/pageflow/ui/views/table_cells/delete_row_table_cell_view.js

```diff
@@ -0,0 +1,28 @@
+pageflow.DeleteRowTableCellView = pageflow.TableCellView.extend({
+  className: 'delete_row_table_cell',
+  template: 'templates/delete_row_table_cell',
+
+  ui: {
+    removeButton: '.remove'
+  },
+
+  events: {
+    'click .remove': 'destroy',
+
+    'click': function() {
+      return false;
+    }
+  },
+
+  isUploading: function() {
+    return this.getModel().isUploading();
+  },
+
+  update: function() {
+    this.ui.removeButton.toggle(!this.isUploading());
+  },
+
+  destroy: function() {
+    this.getModel().destroy();
+  }
+});
```

Comment:
"Leaf views" like this a easy to test. Since this is a highly reusable view, I'd say it's worth it. You can look at the other table cells for examples.